### PR TITLE
zekiel's Wu-Tang members cheat sheet

### PR DESCRIPTION
**Pull request by zekiel**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Shows the members of the Wu-Tang Clan

**What problem does your Instant Answer solve (Why is it better than organic links)?**
When you're trying to remember who's bringing the ruckus -- this will help. 

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://en.wikipedia.org/wiki/Wu-Tang_Clan#Members

**Why did you choose this data source?**
It's accurate

**Are there any other alternative (better) data sources?**
No

**What are some example queries that trigger this Instant Answer?**
wu tang cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
The Bronx

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No
screen shot 2015-06-24 at 2 41 05 pm

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [x] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

------

IA Page: https://duck.co/ia/view/wu_tang_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@zekiel thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/wu-tang.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "id": "wu_tang_cheat_sheet",
+    "name": "Wu-Tang Members",
+    "description": "Members of the Wu-Tang Clan",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/wu-tang.json]: Can probably remove this, or change it to explain that this provides real names and birthdates

File: share/goodie/cheat_sheets/wu-tang.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "key": "GZA"
+        }, {
+            "val": "born Russell Jones, 1968–2004",
+            "key": "Ol' Dirty Bastard"
+        }, {
+            "val": "born Jason Hunter, 1970",
+            "key": "Inspectah Deck"
+        }, {
+            "val": "born Corey Woods, 1970",
+            "key": "Raekwon the Chef"
```
Comment (zekiel) [inline share/goodie/cheat_sheets/wu-tang.json]: Raekwon = Raekwon the Chef 

more appropriate IMO

**Comment by zachthompson** [comment]
@zekiel you trying to stealth in The Game of Thrones Houses cheat sheet or what?

**Comment by zekiel** [comment]
@zachthompson  i facepalmed when I realized what I'd done. Then amused myself at the thought of House Wu-Tang. 

will fix

**Comment by zekiel** [comment]
fixed

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@zekiel congrats! It's live: https://duckduckgo.com/?q=wu+tang+cheat+sheet&ia=cheatsheet

**Pull request merged**